DOC-4389: Add legacy social intranet, the stream and widgets in the D… (#227)
exoplatform/exo-documentation: docs/GettingStarted.rst

       Introduction to what you need to do for the first startup of
       eXo Platform.

:ref:`Navigate on the Digital Workplacce <DWHomepage>`

       Introduction to the navigation on the Digital Workplace, the Digital Workplace

>        applications, Spaces and Settings.

Reviewer: ```suggestion
       widgets, spaces and settings.
```

  .. _DWHomepage:

==============================
Navigate in Digital Workplace
==============================

> This section introduces you to the navigation in the Digital Workplace, ie **Snapshot**. Besides,

Reviewer: ```suggestion
This section introduces the navigation in the Digital Workplace site. Besides,
```

you will further learn about the following topics:
   
 * :ref:`Navigation <Navigation>`
   How to navigate in the Digital Workplace.

 * :ref:`Snapshot <Snapshot>`

>    Introduction to the Digital Workplace Homepage.

Reviewer: ```suggestion
   Introduction to the Snapshot page, all your Digital Workplace at a glance .
```

>  * :ref:`Digital Workplace applications <DW-Gadgets>`

Reviewer: ```suggestion
 * :ref:`Digital Workplace widgets <DW-Gadgets>`
```

>    Introduction to applications that come with the Digital Workplace.

Reviewer: ```suggestion
   Introduction to widget blocks that come with the Digital Workplace.
```

-  *Who's Online?*: See who are online or send your connection request to them. See :ref:`Who's Online? application <WhoIsOnlineApp>` for more details.

.. _Snapshot:

*This section will be developed soon*

> .. _DW-Gadgets:

Reviewer: Widgets

> Digital Workplace applications

Reviewer: ```suggestion
Digital Workplace widgets
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Digital Workplace widgets come with the Digital Workplace, including:

Reviewer: ```suggestion
Digital Workplace comes with several widget applications preinstalled on default pages to provide useful info and handy shortcuts to all apps and key information such as :
```

> Digital Workplace applications are ones which come with the Digital Workplace, including:

Reviewer: ```suggestion
Digital Workplace widgets come with the Digital Workplace, including:
```

you start exploring the Social Intranet by suggesting you where to go
and what you should do first via the following links:

-  :ref:`Set your profile picture <Change-your-avatar>`

-  :ref:`Join a space <Join-space>`

-  :ref:`Add Connections <Send-Connection-Request>`

-  :ref:`Post an activity <Share-AS>`

|DW-1|

Clicking each link will direct you to the related page to do the action.

After each action is performed, it will be remarked as completed with a
check-mark even though it is not performed via this application.

> When all the actions are performed, you can remove this widget from the Stream Page by clicking

Reviewer: ```suggestion
When all the actions are performed, you can dismiss this widget permanently by clicking
```

> When all the actions are performed, you will can remove this application from the Stream Page by clicking

Reviewer: ```suggestion
When all the actions are performed, you can remove this widget from the Stream Page by clicking
```

> Close on the application header|DW-2|.

Reviewer: ```suggestion
Close on the header|DW-2|.
```

>     .. note:: - You can’t get the Getting Started widget back when it is removed.

Reviewer: ```suggestion
    .. note:: - You can’t get the Getting Started widget back after you closed it.
```

>     .. note:: - You cannot get the Getting Started application back when it is removed.

Reviewer: ```suggestion
    .. note:: - You can’t get the Getting Started widget back when it is removed.
```

.. _WelcomeBack:

Welcome Back
-------------

> The welcome back widget is an application that welcomes you with some information about your spaces, connections, 

Reviewer: ```suggestion
The welcome back widget is an block that welcomes you with some information about your spaces, connections, 
```

> your points obtained in the current week and your weekly rank.

Reviewer: ```suggestion
the points you earned and the rank you reached this week.
```

> your points obtained in the current week and your weekly rank.

Reviewer: rank is probably not clear at this stage to a new comer. We should link to the section in gamification where rank concept is described.

Author: Done.

|DW-3|

> - You will see your avatar and your name just after the welcome Back label.

Reviewer: ```suggestion
- You can see your profile picture and your name just after the welcome Back label.
```

I hate the term avatar
turn this sentence to present tense. Future makes no sense when describing an interface.

> - For your spaces, you will see the total number of spaces that you're member of.

Reviewer: ```suggestion
- The widget also displays the  total number of spaces that you're member of.
```

> - For your spaces, you will see the total number of spaces that you're member of, or created.

Reviewer: ```suggestion
- For your spaces, you will see the total number of spaces that you're member of.
```

>   If you have some invitations to join spaces, you will find a badge displaying the number of awaiting invitations

Reviewer: ```suggestion
  A counter of your pending space invitations is displayed in a small badge
```

>   If you have some requests to join a space, you will find a badge displaying the number of requests

Reviewer: ```suggestion
  If you have some invitations to join spaces, you will find a badge displaying the number of awaiting invitations
```

>   you have and you can just click on it to display them in a drawer.

Reviewer: ```suggestion
  Click the badge to display all invitations and easily accept or refuse them.
```

>   you have and you can just click on them to display them in a drawer.

Reviewer: ```suggestion
  you have and you can just click on it to display them in a drawer.
```

>     .. note:: - If you don't have any request to join a space, the badge will be hidden.

Reviewer: ```suggestion
    .. note:: - When you don't have any space invitation, the badge is hidden.
```

> - For your connections, you will see the total number of connections on your network.

Reviewer: ```suggestion
- this widget also reminds you the size of your social network by displaying your current number of Connections.
```

>   If you have some connection requests, you will find a badge displaying the number of requests

Reviewer: ```suggestion
  When you have some pending  connection requests, a small badge displays a requests counter.
```

>   you have and you can just click on them to display them in a drawer.

Reviewer: ```suggestion
 Click the badge to display the list of pending requests and accept or refuse them.
```

>     .. note:: - If you don't have any request to join a space, the badge will be hidden.

Reviewer: ```suggestion
    .. note:: - When you don’t have any connection request, the badge is hidden.
```

> - You can see the points you earned his week and by clicking on your points, you will have a pie chart 

Reviewer: ```suggestion
- You can see the points you earned this week and by clicking on your points, you can see a pie chart 
```

> - You will have your Weekly gamifications gained points and by clicking on your points, you will have a pie chart 

Reviewer: ```suggestion
- You can see the points you earned his week and by clicking on your points, you will have a pie chart 
```

>   that presents your points partition by domains.

Reviewer: ```suggestion
  that presents your points distribution by domain.
```

|DW-4|

> - You can also your rank for this week and by clicking on your rank, you can see a leaderboard 

Reviewer: ```suggestion
- You can also see your rank for this week and by clicking on your rank, you can see a leaderboard 
```

> - You will have also your Weekly rank and by clicking on your rank, you will have a leaderboad 

Reviewer: ```suggestion
- You can also your rank for this week and by clicking on your rank, you can see a leaderboard 
```

>   that presents your position comparing to other members and the amount of points you earned this week so far.

Reviewer: ```suggestion
  that presents your position comparing to other members and how many points you earned this week so far.
```

>   that presents your position comparing to you collaborators and your weekly gained points.

Reviewer: ```suggestion
  that presents your position comparing to other members and the amount of points you earned this week so far.
```

    .. note:: From the pop-up, you can click his avatar or display name to jump to his activity stream page.

.. _Perkstore:

> Perkstore

Reviewer: Perks Store

----------

> The My Orders widget will presents to you the number of your current orders in the Perkstore.

Reviewer: ```suggestion
The Perks Store widget will presents to you the number of your current orders in the Perkstore.
```

> The Perks Store widget will presents to you the number of your current orders in the Perkstore.

Reviewer: ```suggestion
The Perks Store widget displays pending orders you have in the Perks Store application.
```

should add a link to Pers Store doculentztion section

Author: We don't have documentation for perks store, all what i find is a link to github page for Perks sotre.

> By clicking on this number, the Perkstore page will be opened on the list of all your orders.

Reviewer: ```suggestion
By clicking on this number, the Perks Store application will be opened on the list of all your orders.
```

|DW-8|

.. _Wallet:

Wallet Balance
---------------

> The Wallet widget will presents to you the number of your cauris on your wallet.

Reviewer: ```suggestion
The Wallet widget displays the number of your balance of your wallet.
```

> The Wallet widget displays the number of your balance of your wallet.

Reviewer: ```suggestion
The Wallet widget displays the token balance of your wallet.
```

> You will receive tokens regularly in your wallet by collaborating through the digital workplace apps. 

Reviewer: add a see Rewards (link to Reward sextion in the doc)

Author: Done.

> Cauris are earned by the points you gained on gamification when you react and collabore with your collaborators.

Reviewer: ```suggestion
You will receive tokens regularly in your wallet by collaborating through the digital workplace apps. 
```
Should link to the section of the doc that talks about rewards in wallet
